game/spider_sav.ksy

```diff
@@ -0,0 +1,148 @@
+meta:
+  id: spider_sav
+  title: Windows XP Spider Solitaire save file
+  application:
+    - Windows Spider Solitaire
+  file-extension: sav
+  xref:
+    wikidata: Q2512977
+  license: Unlicense
+  endian: le
+  imports:
+    - ../common/random/lcg
```

Comment:
Please use `/common/...` to avoid dependency on position with common components.